index.html

```diff
@@ -267,35 +277,55 @@ <h3 class="ctitletext">
         </h3>
       </div>
       <form class="contactform">
```

Comment:
- [ ] as part of the requirements, your form should uses the action and method attribute to integrate with [formspree service](https://formspree.io/html)